test/topology/conftest.py

```diff
@@ -169,7 +169,9 @@ def fails_without_raft(request, check_pre_raft):
 # "random_tables" fixture: Creates and returns a temporary RandomTables object
 # used in tests to make schema changes. Tables are dropped after finished.
 @pytest.fixture(scope="function")
-def random_tables(request, cql):
+async def random_tables(request, cql, manager):
     tables = RandomTables(request.node.name, cql, unique_name())
     yield tables
+    # NOTE: to avoid occasional timeouts on keyspace teardown, start stopped servers
+    await manager.start_stopped()
     tables.drop_all()
```

Comment:
other options are:
* to let it fail, and ignore it. (the next test uses this cluster would "fix" the state, and clear the tables ?)
* check if any nodes are up, and only then run it.

Reply:
> other options are:
> 
> * to let it fail, and ignore it. (the next test uses this cluster would "fix" the state, and clear the tables ?)
> * check if any nodes are up, and only then run it.

In these topology tests, more often than not clusters are marked dirty and cycled between test cases.